package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -550,18 +570,57 @@ def _read_next_timestep(self):
             alpha = beta = gamma = 90.
         ts.dimensions = xlen, ylen, zlen, alpha, beta, gamma
 
+        _get_column = lambda ids, col_ids : [col_ids[i] for i in ids if
+                                             all(id in col_ids.keys() for
+                                             id in ids)]
+                                             
         indices = np.zeros(self.n_atoms, dtype=int)
 
-        f.readline()  # ITEM ATOMS etc
-        for i in range(self.n_atoms):
-            idx, _, xs, ys, zs = f.readline().split()
+        atomline = f.readline()  # ITEM ATOMS etc
+        attrs = atomline.split()[2:]  # attributes on coordinate line
+        col_ids = {attr: i for i, attr in enumerate(attrs)}  # column ids
+        keys = col_ids.keys()
+        # check for ids and what type of coordinate convention
+        ids = "id" in keys
+        # keys = convention values = col_ids or False if not present
+        coord_dict = {conv: False for conv in self._conventions[1:]}
+        coord_dict['unscaled'] = _get_column(
```

Comment:
Will `coord_dict` contain the column numbers? Perhaps add a comment to explain the parsing logic.